debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -188,6 +194,79 @@ public class JdbcSinkConnectorConfig implements SinkConnectorConfig {
             .withDefault(DefaultColumnNamingStrategy.class.getName())
             .withDescription("Name of the strategy class that implements the ColumnNamingStrategy interface.");
 
+    public static final Field COLUMN_NAMING_STYLE_FIELD = Field.create(COLUMN_NAMING_STYLE)
+            .withDisplayName("Column Naming Style")
+            .withType(Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("default")
+            .withDescription("The style of column naming: snake_case, camelCase, kebab-case, UPPERCASE, lowercase.");
+
+    public static final Field COLUMN_NAMING_PREFIX_FIELD = Field.create(COLUMN_NAMING_PREFIX)
```

Comment:
What do you think about adding empty string defaults for the prefix/suffix? I believe this could simplify the checks in other places because these properties would either be a populated or empty string but never `null`.  wdyt?

Reply:
See my other previous comment. On similar use-case we use `collection.name.format` to add an optional prefix and/or suffix. Should we keep this behavior instead of adding a new way by adding 2 options (prefix/suffix)? That would align better with how we do this and it allows combining prefix and suffix in a single option.

Reply:
Hi @rk3rn3r, that's certainly another way to tackle it, though we'd need to introduce a placeholder for the name of where it should be injected as we did with `${topic}` and `${source.table}` for the collection name.  I'm not partial to either approach.